### PR TITLE
Update DM TOC and remove redundant files

### PR DESCRIPTION
**Pull request by shichun-0415**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
Update DM TOC and remove redundant files

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/8119
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: migration-overview.md [hunk 1]
```diff
@@ -56,6 +56,6 @@ If the data size of the sharded tables is large (for example, larger than 1 TiB)
 The following features can improve the migration process and might meet more needs in your application.
 
 - [Migrate with pt/gh-host](/migrate-with-pt-ghost.md)
-- [Migrate with Binlog Event Filter](/filter-binlog-event.md)
-- [Migrate with Filter Binlog Events Using SQL Expressions](/filter-dml-event.md)
 - [Migrate with More Columns in Downstream](/migrate-with-more-columns-downstream.md)
+- [Migrate with Binlog Event Filter](/filter-binlog-event.md)
+- [Migrate with Filter Binlog Events Using SQL Expressions](/filter-dml-event.md)
```
Comment (sunzhaoyang) [inline migration-overview.md:61]:
```suggestion
- [Migrate with DMLs Using SQL Expressions](/filter-dml-event.md)
```

**Review by sunzhaoyang — COMMENTED** [review]
rest LGTM

File: migration-overview.md (hunk 1 above)
Comment (shichun-0415) [reply migration-overview.md]: Thanks. Reference links are fixed in https://github.com/pingcap/docs/pull/7493#discussion_r794241003.

File: dm/feature-online-ddl.md [hunk 2]
```diff
@@ -6,66 +6,9 @@ aliases: ['/docs/tidb-data-migration/dev/online-ddl-scheme/','tidb-data-migratio
...
-  host: "192.168.0.1"
-  port: 4000
-  user: "root"
-  password: ""                  # It is recommended to use password encrypted with dmctl if the password is not empty.
-```
-
-For the advanced configuration and the description of each configuration parameter, refer to [DM advanced task configuration file template](/dm/task-configuration-file-full.md#task-configuration-file-template-advanced).
-
-When you merge and migrate data from sharded tables, you need to coordinate the DDL of each sharded table, and the DML before and after the DDL. DM supports two different modes: pessimistic mode and optimistic mode. For the differences and scenarios between the two modes, refer to [Merge and Migrate Data from Sharded Tables](/dm/feature-shard-merge.md).
+When using DM to migrate data from MySQL to TiDB, you can enbale online-ddl to allow collaboration of DM and gh-ost or pt-osc. For details about how to enable online-ddl and the workflow after enabling this option, see [Continuous Replication with gh-ost or pt-osc](/migrate-with-pt-ghost.md). This document focuses on the collaboration details of DM and online DDL tools.
```
Comment (hfxsd) [inline dm/feature-online-ddl.md:11]: 中文在这段话上面有个标题 “## DM 与 online DDL 工具协作细节”，请确认下是否需要翻译。

File: dm/quick-create-migration-task.md [hunk 3]
```diff
@@ -12,25 +12,49 @@ summary: Learn how to configure a data migration task in different scenarios.
...
+When you migrate data from Aurora to a TiDB cluster deployed on AWS, your data migration takes two operations: full data migration and incremental replication. You can choose the corresponding operation according to your application needs.
 
-## Migrate Sharded Schemas and Sharded Tables to TiDB
+-[Migrate Data from Amazon Aurora to TiDB](/migrate-aurora-to-tidb.md)
 
-If you need to migrate sharded schemas and sharded tables to TiDB, refer to [Data Migration Shard Merge Scenario](/dm/usage-scenario-shard-merge.md).
+## Migrate from MySQL to TiDB
 
-## Migrate Incremental Data to TiDB
+If cloud storage (S3) service is not used, the network connectivity is good, and the network latency is low, you can use the following method to migrate data from MySQL to TiDB.
```
Comment (hfxsd) [inline dm/quick-create-migration-task.md:30]:
```suggestion
If a cloud storage (S3) service is not used, the network connectivity is good, and the network latency is low, you can use the following method to migrate data from MySQL to TiDB.
```

**Review by hfxsd — APPROVED** [review]
Rest LGTM

File: TOC.md [hunk 4]
```diff
@@ -250,19 +250,14 @@
...
       - [Quick Start](/dm/quick-start-with-dm.md)
       - [Deploy a DM cluster Using TiUP](/dm/deploy-a-dm-cluster-using-tiup.md)
       - [Create a Data Source](/dm/quick-start-create-source.md)
-      - Data Migration Scenarios
-        - [Data Migration Scenario Overview](/dm/quick-create-migration-task.md)
-        - [Migrate Data from Multiple Data Sources to TiDB](/dm/usage-scenario-simple-migration.md)
-        - [Migrate Sharded Schemas and Tables to TiDB](/dm/usage-scenario-shard-merge.md)
-        - [Migrate Incremental Data to TiDB](/dm/usage-scenario-incremental-migration.md)
-        - [Migrate Tables when There Are More Columns Downstream](/dm/usage-scenario-downstream-more-columns.md)
+      - [Data Migration Scenarios](/dm/quick-create-migration-task.md)
```
Comment (qiancai) [inline TOC.md:260]: According to the Chinese PR https://github.com/pingcap/docs-cn/pull/8119, this section needs to be at the same level as Quick Start.

File: dm/feature-expression-filter.md [hunk 5]
```diff
@@ -1,91 +1,14 @@
...
+aliases: ['tidb/dev/feature-expression-filter/']
 ---
 
-# Filter Certain Row Changes Using SQL Expressions
+# Filter DMLs Using SQL Expressions
 
 ## Overview
 
-In the process of data migration, DM provides the [Binlog Event Filter](/dm/dm-key-features.md#binlog-event-filter) feature to filter certain types of binlog events. For example, for archiving or auditing purposes, `DELETE` event might be filtered when data is migrated to the downstream. However, Binlog Event Filter cannot judge with a greater granularity whether the `DELETE` event of a certain row should be filtered.
+In the process of incremental data migration, you can filter certain types of binlog events using the [Filter Binlog Events](/filter-binlog-event.md) feature. For example, for archiving or auditing purposes, `DELETE` event might be filtered when data is migrated to the downstream. However, Binlog Event Filter cannot judge with a greater granularity whether the `DELETE` event of a certain row should be filtered.
```
Comment (qiancai) [inline dm/feature-expression-filter.md:10]:
```suggestion
In the process of incremental data migration, you can filter certain types of binlog events using the [Filter Binlog Events](/filter-binlog-event.md) feature. For example, for archiving or auditing purposes, you can filter out `DELETE` events when migrating data to the downstream. However, Binlog Event Filter cannot judge with a greater granularity on whether to filter out a specific row of `DELETE` events.
```

File: dm/quick-create-migration-task.md [hunk 6]
```diff
@@ -12,25 +12,49 @@ summary: Learn how to configure a data migration task in different scenarios.
 > 1. [Deploy a DM Cluster Using TiUP](/dm/deploy-a-dm-cluster-using-tiup.md).
 > 2. [Create a Data Source](/dm/quick-start-create-source.md).
 
-This document introduces how to configure a data migration task in different scenarios. You can choose suitable documents to create your data migration task according to the specific scenario.
+This document introduces how to perform data migration tasks in different business scenarios.
```
Comment (qiancai) [inline dm/quick-create-migration-task.md:15]:
```suggestion
This document introduces how to perform data migration tasks in different scenarios.
```

File: dm/quick-create-migration-task.md [hunk 7]
```diff
@@ -12,25 +12,49 @@ summary: Learn how to configure a data migration task in different scenarios.
...
+If the data size of the sharded tables is large (for example, larger than 1 TiB), and you do not allow other applications to write to TiDB during the migration period, you can use TiDB Lightning to quickly merge and import the sharded tables. Then, you can use DM to replicate incremental sharding data (binlog) based on your application needs.
+
+- [Migrate and Merge MySQL Shards of Large Datasets to TiDB](/migrate-large-mysql-shards-to-tidb.md)
+
+## Migrate data from files to TiDB
+
+- [Migrate data from CSV files to TiDB](/migrate-from-csv-files-to-tidb.md)
+- [Migrate data from SQL files to TiDB](/migrate-from-sql-files-to-tidb.md)
+
+## More complex migration solutions
```
Comment (qiancai) [inline dm/quick-create-migration-task.md:53]:
```suggestion
## More advanced migration solutions
```

Comment (qiancai) [reply dm/quick-create-migration-task.md:53]: As a user, I might not hope the any product solution is complex

File: dm/quick-create-migration-task.md [hunk 8]
```diff
@@ -12,25 +12,49 @@ summary: Learn how to configure a data migration task in different scenarios.
...
+
+The following features can improve the migration process and might meet more needs in your application.
```
Comment (qiancai) [inline dm/quick-create-migration-task.md:55]:
```suggestion
The following features can improve the migration process and meet more needs in your application.
```

**Review by qiancai — COMMENTED** [review]
@shichun-0415  I've complete the review and left some comments. PTAL

File: dm/feature-online-ddl.md (hunk 2 above)
Comment (shichun-0415) [reply dm/feature-online-ddl.md:11]: 是指下面第 13 行的标题吗？

Comment (hfxsd) [reply dm/feature-online-ddl.md:11]: OK 是的 就是这个

File: dm/quick-create-migration-task.md [hunk 9]
```diff
@@ -12,25 +12,49 @@ summary: Learn how to configure a data migration task in different scenarios.
...
+## Migrate Data from Amazon Aurora to TiDB
 
-If you need to migrate data from multiple data sources to TiDB, and to rename tables to avoid migration conflicts caused by duplicate table names in different data sources, or to disable some DDL/DML operations in some tables, refer to [Migrate Data from Multiple Data Sources to TiDB](/dm/usage-scenario-simple-migration.md).
+When you migrate data from Aurora to a TiDB cluster deployed on AWS, your data migration takes two operations: full data migration and incremental replication. You can choose the corresponding operation according to your application needs.
 
-## Migrate Sharded Schemas and Sharded Tables to TiDB
+-[Migrate Data from Amazon Aurora to TiDB](/migrate-aurora-to-tidb.md)
 
-If you need to migrate sharded schemas and sharded tables to TiDB, refer to [Data Migration Shard Merge Scenario](/dm/usage-scenario-shard-merge.md).
+## Migrate from MySQL to TiDB
```
Comment (qiancai) [inline dm/quick-create-migration-task.md:28]:
```suggestion
## Migrate data from MySQL to TiDB
```

File: dm/quick-create-migration-task.md [hunk 10]
```diff
@@ -12,25 +12,49 @@ summary: Learn how to configure a data migration task in different scenarios.
...
+This document introduces how to perform data migration tasks in different scenarios.
 
-In addition to scenario-based documents, you can also refer to the following ones:
+In addition to scenario-based documents, you can also refer to the following documents:
 
 - For a complete example of data migration task configuration, refer to [DM Advanced Task Configuration File](/dm/task-configuration-file-full.md).
 - For a data migration task configuration guide, refer to [Data Migration Task Configuration Guide](/dm/dm-task-configuration-guide.md).
 
-## Migrate Data from Multiple Data Sources to TiDB
+## Migrate Data from Amazon Aurora to TiDB
```
Comment (qiancai) [inline dm/quick-create-migration-task.md:22]:
```suggestion
## Migrate data from Amazon Aurora to TiDB
```

**Review by qiancai — APPROVED** [review]
LGTM

**Pull request merged**